docs/develop.md

```diff
@@ -26,9 +26,14 @@ make tilt-up
 
 You should make sure that everything in the UI looks green. If not, you can trigger the Tilt workflow again. In the case of the `cspotemplate`, this might be necessary, as it cannot be applied right after the startup of the cluster and fails. Tilt unfortunately does not include a waiting period.
```

Comment:
`If not, you can trigger the Tilt workflow again. In the case of the `cspotemplate`, this might be necessary, as it cannot be applied right after the startup of the cluster and fails. Tilt unfortunately does not include a waiting period.`

IMO this statement is not true, because this PR removed auto-deployment of cspotemplate